### ISSUE TITLE
support dbuild (Scala community distributed build)

### ISSUE THREAD

**Issue by SethTisue** [NetLogo/Tortoise#71]
it would be nice to be included in the community build process since we'd get early notification of bugs and incompatibilities with forthcoming Scala and sbt versions.

on Twitter, @gkossakowski writes:

> @SethTisue have you considered using bintray-sbt plugin instead of your own plugin here: https://github.com/NetLogo/Tortoise/blob/master/project/plugins.sbt#L16 … ?
> 
> @sethtisue yes, custom repositories are not very cool if we want to add a project to dbuild. The problem is artifact resolution performance
> 
> @sethtisue I'm trying to give tortoise a spin by hacking repository definitions. That's a one-off, of course.
> 
> @sethtisue The result of compiling tortoise is: https://gist.github.com/gkossakowski/9609683 … Good/bad/neutral?
> 
> ```
> ```

**Comment by SethTisue** [comment]
perhaps ironically, I am now working on a Java 8 based community build myself.

**Comment by mrerrormessage** [comment]
@SethTisue , I would be *very* interested in getting this enabled, especially since this is a reasonably large project compiling in both scala and scala.js, and using features like macros. I would also love to have headless as part of this, and it might make a better candidate as I believe it has all dependencies on Maven Central. I would happily put in some work to get the build selectively enabling/disabling plugins, and I think it would be more than worth the effort for headless and/or tortoise.

**Comment by SethTisue** [comment]
I've drafted a new writeup on community build eligibility: https://github.com/scala/community-builds/wiki

TL;DR: we're not really able to accept projects like NetLogo right now, sadly.
